docs/FAQ.md

```diff
@@ -0,0 +1,125 @@
+# Frequently Asked Questions
+
+If you can't find the answer to your question here, please ask us in [Discord](https://discord.com/channels/752978124614008945).
+
+
+## General
+
+### How is the Yield Protocol monetized?
+The Yield company doesn't have a monetization plan yet. All fees in the liquidity pools go to liquidity providers.
+
+### How can I find you on uniswap?
+There is no governance token. You will not find us on Uniswap. You could find the fyDai or fyDaiLP tokens on Uniswap or other exchanges, but you should get better prices for them in [Yield](app.yield.is) since the app is tailored to the particularities of trading fyDai, such as its maturity date.
+
+### When will you release a governance token?
+There are no current plans to release a governance token.
+
+### How is APR calculated?
+
+(getFyDaiReserves / getDaiReserves)^(1/4) - 1 should be APR (3 months to maturity)
+pool.sellFYDaiPreview(1)**(time to maturity in years) - 1
+
+### What is the fee structure? Formula/parameters?
+The fee structure is detailed in our [YieldSpace paper](https://yield.is/YieldSpace.pdf). See Section 5.
+
+### When telegram?
+We have a thriving community in [Discord](https://discord.com/channels/752978124614008945), join us there!
+
+### Does MakerDAO have to have support for all assets that can be listed on yield?
+Yield only works for Dai for v1 and only accepts ETH collateral
+
+### Is this built with UMA?
+Yield is not built with UMA. It is built on top of Maker
+
+### Are there any major differences between fyDai and UMA's uUSD?
+The two main differences are the collateral asset and the pool mechanics.
+
+In UMA you can use any asset approved by the UMA governance, while in Yield you will use Dai for Lending and Liquidity or Eth for Borrowing. Post maturity on uUSD you will have exposure to ETH, while in Yield you will have exposure to Dai.
```

Comment:
Actually the collateral assets are uUSDrBTC uses renBTC and uUSDwETH uses WETH

Reply:
It's written in L47 below